packages/ui/src/components/MessageInput/index.native.tsx

```diff
@@ -199,6 +199,29 @@ export function MessageInput({
                 }
 
               });
+
+              function updateContentHeight() {
+                const editorElement = document.querySelector('#root div .ProseMirror');
+                editorElement.style.height = 'auto';
+                editorElement.style.overflow = 'auto';
+                const newHeight = editorElement.scrollHeight;
+                window.ReactNativeWebView.postMessage(JSON.stringify({ type: 'contentHeight', payload: newHeight }));
+              }
+
+              function setupMutationObserver() {
+                // watch for changes in the content
+
+                const observer = new MutationObserver(updateContentHeight);
+                observer.observe(document.querySelector('#root'), { childList: true, subtree: true, attributes: true});
+
+                updateContentHeight(); // this sets initial height
+              }
+
+              // wait for the editor to load
+              setTimeout(() => {
+               setupMutationObserver();
+              }, 300);
```

Comment:
Any way to wire this to a `ready` event or something like that? Timeouts just intrinsically flakey.

Reply:
Yeah we can send a ready event from the WebView side